string_manipulation/helper.py

```diff
@@ -0,0 +1,53 @@
+
+
+def remove_multy_occurence(multy_occuring_string):
```

Comment:
multi* typo
is this for removing first occurrence or last occurrence? Better naming conventions must be used.